syntax/ebuild.vim

```diff
@@ -372,7 +382,9 @@ hi def link EbuildCDROMKeyword               Identifier
 hi def link EbuildLinuxInfoKeyword           Identifier
 hi def link EbuildUnpackerKeyword            Identifier
 hi def link EbuildUserKeyword                Identifier
+hi def link EbuildCMakeKeyword               Identifier
 hi def link EbuildDistutilsFunction          Special
+hi def link EbuildCMakeFunction              Special
```

Comment:
This doesn't seem correct. `EbuildDistutilsFunction` is about functions that are defined in ebuilds, whereas `EbuildCMakeFunction` seem to be just regular phases.

Reply:
OK i switched these to `EbuildCMakeKeyword`